grammar.js

```diff
@@ -25,6 +25,6 @@ module.exports = grammar({
 
     expression: ($) => seq(ANYTHING, NEWLINE),
 
-    result: ($) => token(prec(-1, ANYTHING)),
+    result: ($) => prec.right(repeat1(seq(token(prec(-1, ANYTHING)), NEWLINE))),
```

Comment:
Ahh sure that's a generalization of what we had, great!

There is an edge case like this:

```
iex> x
1<no newline here>
```

but we can likely use `sep1` instead of `repeat`

```js
function sep1(rule, separator) {
  return seq(rule, repeat(seq(separator, rule)));
}
```